website/docs/cli/commands/state/show.mdx

```diff
@@ -68,8 +68,7 @@ $ terraform state show 'packet_device.worker[0]'
 
 ## Example: Show a Resource configured with for_each
 
-The example below shows the `"example"` instance of a `packet_device` resource named `worker` configured with
-[`for_each`](/language/meta-arguments/for_each):
+The following example shows the `"example"` instance of a packet_device resource named worker configured with [`for_each`](/language/meta-arguments/for_each). You must place the resource name in single quotes when it contains special characters (e.g., double quotes).
```

Comment:
```suggestion
The following example shows the `"example"` instance of a `packet_device` resource named `worker` configured with the [`for_each`](/language/meta-arguments/for_each) meta-argument. You must place the resource name in single quotes when it contains special characters (e.g., double quotes).
```

Reply:
@irab One last thing! We recently decided in our style guide that we don't want to include "e.g." anymore - we spell it out fully or replace it with an alternative. This makes the text easier to read. I also made some opportunistic fixes because I realized that we needed to put the resource name and type in code font, as per the original example. 

I promise this is the final edit, and then we'll merge! Thank you for your patience with this.